specs/protocol/derivation.md

```diff
@@ -1014,6 +1017,12 @@ entries.
 
 [deposit-contract-spec]: deposits.md#deposit-contract
 
+Logs for all transaction types are parsed for deposits, also from possibly unknown transactions types like those that
+are considered invalid batcher transaction types, see [L1 Retrieval][#l1-retrieval]. However, they are parsed as if they
+were receipts for one of the known transactions types `0`, `1`, `2`, `3`, `4` and `0x7e`. If they fail this best-effort
+decoding process, they are dropped and considered invalid.
+The intention of this best-effort decoding is to future-proof the protocol for new L1 transaction types.
```

Comment:
I'm not sure if this "best-effort" specification is too vague and not precise enough. It seems like this could be too implementation-dependent...